service/build.gradle

```diff
@@ -224,7 +227,8 @@ dependencies {
 
   compile group: 'org.springframework.boot', name: 'spring-boot-starter-mail'
 
-  testImplementation group: 'org.apache.pdfbox', name: 'pdfbox', version: '2.0.20'
+  testImplementation group: 'org.apache.pdfbox', name: 'pdfbox', version: versions.pdfbox
+  integrationTestImplementation group: 'org.apache.pdfbox', name: 'pdfbox', version: versions.pdfbox
```

Comment:
tbh, dont understand why this is needed, but yes, checked and tests fails without it, something is messed up, as it should not be needed

Reply:
I also don't understand 😄 discussed with @lewisbirks and this was what we found fixed the failing tests